src/components/AutoInputBinLocation/index.tsx

```diff
@@ -0,0 +1,39 @@
+import React, {useState} from 'react';
+import styles from './styles';
+import {View, TextInput} from 'react-native';
+import {Props} from './types';
+import ModalSelector from 'react-native-modal-selector-searchable';
+
+export default function ({data, selectedData, initValue = ''}: Props) {
```

Comment:
How does this component differ from `AutoInputInternalLocation`? Why this one was created? If you just wanted rename from `internal` to `bin` then why the `internal` one was not removed?

Reply:
Sorry, I requested that we create a new autocomplete input component that would allow us to select Bin Locations instead of Receiving Locations. I wasn't sure the best approach but I figured each of these components would extend a common component that allowed us to customize the API request / response mechanism.

Eventually we're going to need an autocomplete for products, inventory items, locations, reason codes, receiving locations, bin locations, all internal locations, etc. Some will be able to load all values from the server e.g. reason codes. But others will need to pull data based on a search query (cachine and/or limiting the results) for APIs that return large amounts of data like the Products API.